src/open/clients/Quaternion.js

```diff
@@ -20,8 +20,8 @@ export class Quaternion {
     get id() { return "quaternion"; }
     get name() { return "Quaternion"; }
     get icon() { return "images/client-icons/quaternion.svg"; }
-    get author() { return "Felix Rohrbach"; }
-    get homepage() { return "https://github.com/Fxrh/Quaternion"; }
+    get author() { return "Felix Rohrbach and Alexey Rusakov "; }
```

Comment:
```suggestion
    get author() { return "The Quotient project"; }
```
I'm not sure where it's exposed in the UI but I guess it's most sustainable to use the collective name in this case.